packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -452,9 +481,19 @@ export function getDestinationChainIds(chainId: ChainId): ChainId[] {
     ? arbitrumSdkChain.partnerChainID
     : undefined
 
-  const validDestinationChainIds =
+  let validDestinationChainIds =
```

Comment:
can const + push

Reply:
+1

Reply:
This one is still let. We do something like this: https://github.com/OffchainLabs/arbitrum-token-bridge/pull/1418#discussion_r1603343625